var/spack/repos/builtin/packages/plasma/package.py

```diff
@@ -0,0 +1,82 @@
+##############################################################################
+# Copyright (c) 2017, Innovative Computing Laboratory
+# Produced at the Innovative Computing Laboratory.
+#
+# Created by Piotr Luszczek, luszczek@icl.utk.edu, All rights reserved.
+#
+# For details, see https://github.com/llnl/spack
+#
+##############################################################################
+#
+from spack import *
+
+
+class Plasma(MakefilePackage):
+    """Parallel Linear Algebra Software for Multicore Architectures, PLASMA is
+    a software package for solving problems in dense linear algebra using
+    multicore processors and Xeon Phi coprocessors. PLASMA provides
+    implementations of state-of-the-art algorithms using cutting-edge task
+    scheduling techniques. PLASMA currently offers a collection of routines for
+    solving linear systems of equations, least squares problems, eigenvalue
+    problems, and singular value problems."""
+
+    homepage = "https://bitbucket.org/icl/plasma/"
+    url      = "https://bitbucket.org/icl/plasma/downloads/plasma-17.1.tar.gz"
+
+    version("17.1", "64b410b76023a41b3f07a5f0dca554e1")
+
+    version("develop", hg="https://luszczek@bitbucket.org/icl/plasma")
+
+    depends_on("blas")
+    depends_on("lapack")
+
+    conflicts("atlas")  # does not have LAPACKE interface
+    conflicts("netlib-lapack@:2.999")  # missing LAPACKE features
+    # clashes with OpenBLAS declarations and has a problem compiling on its own
+    conflicts("cblas")
+    conflicts("openblas-with-lapack")  # incomplete LAPACK implementation
+    conflicts("veclibfort")
+
+    conflicts("gcc@:6")  # support for OpenMP 4+ is required, available in 7.0
```

Comment:
Isn't that case covered by `conflicts("%gcc@3.0:6.999")` below?

Reply:
Yes, conflicts("gcc...") and conflicts("%gcc...") are redundant. I killed one.